src/Simplex/Chat/Migrations/M20230922_remote_hosts.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Migrations.M20230922_remote_hosts where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20230922_remote_hosts :: Query
+m20230922_remote_hosts =
+  [sql|
+CREATE TABLE remote_hosts (
+  zone_id INTEGER PRIMARY KEY,
+  display_name TEXT NOT NULL,
+  path TEXT NOT NULL,
+  ca_key BLOB NOT NULL,
+  ca_cert BLOB NOT NULL
```

Comment:
probably not all fields that are needed. We normally store public key separately, so you don't need to read private key when only public is needed, and the naming should be ca_private_key. Also you need the second key and cert, not to change TLS stack that expects 2-cert chain (with the fingerprint from the root).

Reply:
I didn't encounter the need for public key. This pair only used to derive an session certificate which is ephemeral.